AdminApp/src/app/flight-adding/flight-adding.component.html

```diff
@@ -0,0 +1,111 @@
+<div class="infoContainer">
```

Comment:
Please maintain the same indent style throughout the code base. You were using 2 spaces indent in html files before, why this file now has 4?